src/snyk/analytics/itly.ts

```diff
@@ -114,10 +114,8 @@ export class Iteratively {
       return;
     }
 
-    const userId = this.userId ?? '';
-
     itly.welcomeIsViewed(
-      userId,
+      '',
```

Comment:
I guess this is fix for merge-itly-anonymous-user story, right?

Reply:
@pavel-snyk i've just removed the redundant code because `this.userId` will never be set at this point, as this event is fired only when a user is not authenticated.